assets/app/scripts/app.js

```diff
@@ -8,6 +8,7 @@
  *
  * Main module of the application.
  */
+var OPENSHIFT_BUILT_TABS = [];
```

Comment:
yeah i think i can do this with something like angular.module('openshiftConsole').value("builtTabs", {})

will try it